playwright-e2e/dsm/pages/participant-page/rgp/rgp-participant-page.ts

```diff
@@ -70,8 +71,8 @@ export default class RgpParticipantPage extends ParticipantPage {
                 const selectedRelation = await this.relation.toLocator().locator('.mat-select-value-text').innerText();
                 await this.submit.click();
 
-                await expect(page.locator(modal)).toHaveText('Successfully added family member');
-                await page.locator('h1').click(); // close popup with a click outside
+                await expect(page.locator(familyMemberAddedSuccessfullyModal)).toHaveText('Successfully added family member');
```

Comment:
made change since once it turns into the 'success' modal, it seems that for a split second there are 2 webelements that fit role=dialog